src/boilerplate/orchestration/javascript/raw/toOrchestration.ts

```diff
@@ -497,10 +497,8 @@ export const OrchestrationCodeBoilerPlate: any = (node: any) => {
       node.returnParameters.forEach( (param, index) => {
        if(param === 'true')
         rtnparams?.push('bool: bool');
-       else if(param?.includes('Commitment'))
-        rtnparams?.push( ` ${param} : ${param}.integer  `);
        else
-        rtnparams.push(`   ${param} :${param}.integer`);
+       rtnparams?.push( ` ${param}_newCommitmentValue : ${param}.integer  `);
```

Comment:
Maybe unknown variables could look different to this? For example increment might be _newCommitmentValue and decrement might be _change. (This might be included in param but just checking)